Assignment-3/data-cleaning

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+import pandas as pd
+#%matplotlib notebook
+%matplotlib inline
+import matplotlib.pyplot as plt
+from IPython.core.debugger import set_trace
+import warnings
+warnings.filterwarnings('ignore')
+import os
+
+from typing import List
+from tqdm import tqdm
+
+# Assuming the dataset was downloaded from Kaggle, tar file unzipped,
+# and is on your Desktop in a file called archive
+
+# Set up the home directory
+home_directory = os.path.expanduser("~")
+
+# File paths for the dataset
+train_file_path = os.path.join(home_directory, 'Desktop/archive/sign_mnist_train.csv')
+test_file_path = os.path.join(home_directory, 'Desktop/archive/sign_mnist_test.csv')
+
+# Load the datasets
+train_df = pd.read_csv(train_file_path)
+test_df = pd.read_csv(test_file_path)
+
+# Vectorize data, and  normalize
+
+x_train = train_df.iloc[:, 1:].values / 255.0
+x_test = test_df.iloc[:, 1:].values / 255.0
+
+# Reshape the pixel values into flat vectors
+x_train = x_train.reshape(-1, 28*28)
```

Comment:
Can we name this into `x_train_mlp`? The CNN model takes a different dimension of input so it'd be easier if we can just keep the original variable as `x_train` and then reshape it into`x_train_mlp` and `x_train_cnn` when needed